docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/rdb-loader-previous-versions/rdb-loader-4-0-x/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "RDB Loader 4.0.x"
```

Comment:
I notice this page: https://deploy-preview-139--snowplow-docs.netlify.app/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/rdb-loader-previous-versions/

doesn't have the card view on it. e.g. https://raw.githubusercontent.com/snowplow/documentation/1a830685de520cc937814bb911ab85aef8e49212/docs/try-snowplow/recipes/index.md